modules/ROOT/pages/api-groups-add-sla-tiers.adoc

```diff
@@ -8,28 +8,26 @@ You can add multiple SLA tiers to an API Group instance.
 
 To add an SLA tier:
 
-. From the left menu in *API Manager*, click *API Group*.
+. From the left menu in *API Manager*, click *API Groups*.
```

Comment:
For accessibility, can you say this in a different way to avoid using "left"?

Reply:
ah, good catch - will fix